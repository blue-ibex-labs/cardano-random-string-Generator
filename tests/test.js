import React from "react";
import renderer from "react-test-renderer";
import MintingForm from "../pages/form";

describe("Jest Snapshot testing suite", () => {
  it("Matches DOM Snapshot", () => {
    const domTree = renderer.create(<MintingForm />).toJSON();
    expect(domTree).toMatchSnapshot();
  });
});
