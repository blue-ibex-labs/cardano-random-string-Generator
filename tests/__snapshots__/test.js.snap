// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest Snapshot testing suite Matches DOM Snapshot 1`] = `
<div
  className="py-10 bg-gray-100"
>
  <div
    className="flex items-center justify-center p-12"
  >
    <div
      className="mx-auto w-full max-w-[550px] bg-white"
    >
      <form
        className="py-6 px-9"
      >
        <label
          className="mb-3 block text-base font-medium text-[#07074D]"
        >
          Connect Your Wallet
        </label>
        <label
          className="mb-3 mt-3 block text-base font-medium text-[#07074D]"
        >
          * are required
        </label>
        <div
          className="form-group"
        >
          <input
            className="w-full rounded-md border border-[#e0e0e0] bg-white my-3 py-3 px-6 text-base font-medium text-[#6B7280] outline-none focus:border-[#6A64F1] focus:shadow-md"
            name="name"
            placeholder="Project Name"
            type="text"
          />
        </div>
        <div
          className="form-group"
        >
          <input
            className="w-full rounded-md border border-[#e0e0e0] bg-white my-3 py-3 px-6 text-base font-medium text-[#6B7280] outline-none focus:border-[#6A64F1] focus:shadow-md"
            name="url"
            placeholder="Project URL"
            type="text"
          />
        </div>
        <div
          className="form-group"
        >
          <input
            className="w-full rounded-md border border-[#e0e0e0] bg-white my-3 py-3 px-6 text-base font-medium text-[#6B7280] outline-none focus:border-[#6A64F1] focus:shadow-md"
            name="password"
            placeholder="Enter your password"
            type="password"
          />
        </div>
        <div
          className="form-group"
        >
          <input
            className="w-full rounded-md border border-[#e0e0e0] bg-white my-3 py-3 px-6 text-base font-medium text-[#6B7280] outline-none focus:border-[#6A64F1] focus:shadow-md"
            name="Description"
            placeholder="Description"
            type="text"
          />
        </div>
        <div
          className="form-group"
        >
          <input
            className="w-full rounded-md border border-[#e0e0e0] bg-white my-3 py-3 px-6 text-base font-medium text-[#6B7280] outline-none focus:border-[#6A64F1] focus:shadow-md"
            name="Twitter"
            placeholder="Twitter"
            type="text"
          />
        </div>
        <div
          className="form-group"
        >
          <input
            className="w-full rounded-md border border-[#e0e0e0] bg-white my-3 py-3 px-6 text-base font-medium text-[#6B7280] outline-none focus:border-[#6A64F1] focus:shadow-md"
            name="Discord"
            placeholder="Discord"
            type="text"
          />
        </div>
        <div
          className="form-group"
        >
          <input
            className="w-full rounded-md border border-[#e0e0e0] bg-white my-3 py-3 px-6 text-base font-medium text-[#6B7280] outline-none focus:border-[#6A64F1] focus:shadow-md"
            name="Discord"
            placeholder="Discord"
            type="file"
          />
        </div>
        <button
          className="hover:shadow-form w-full rounded-md bg-[#6A64F1] py-3 px-8 text-center text-base font-semibold text-white outline-none"
          onClick={[Function]}
        >
          Submit
        </button>
      </form>
    </div>
  </div>
</div>
`;
